docs/architecture/README.md

```diff
@@ -36,6 +36,12 @@ If recorded decisions turned out to be lacking, convene a discussion, record the
 
 Read about the [PROCESS](./PROCESS.md).
 
+#### Use RFC 2119 Keywords
+
+In many documents several words are used to signify the requirements in the specification.  These words are often capitalized: "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL. They are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+
+
```

Comment:
should'nt the markdown linter catch these?

```suggestion
```

Reply:
This should be fixed with `make lint-fix`. Maybe we should be automatically running this in the CI?